src/store/peer/reducer.ts

```diff
@@ -55,12 +66,24 @@ const deletedPeer = createReducer<DeleteResponse<string | null>, ActionTypes>(in
     .handleAction(actions.deletedPeer.success, (store, action) => action.payload)
     .handleAction(actions.deletedPeer.failure, (store, action) => action.payload)
     .handleAction(actions.setDeletePeer, (store, action) => action.payload)
+    .handleAction(actions.resetDeletedPeer, () => initialState.deletedPeer)
```

Comment:
Codacy has a fix for the issue: [Replace `····.handleAction(actions.resetDeletedPeer,·()·=>·initialState.deletedPeer)` with `··.handleAction(actions.resetDeletedPeer,·()·=>·initialState.deletedPeer);`](https://app.codacy.com/gh/netbirdio/dashboard/pullRequest?prid=9560373)
```suggestion
  .handleAction(actions.resetDeletedPeer, () => initialState.deletedPeer);
```